src/onboarding/onboarding.js

```diff
@@ -1,4 +1,10 @@
 function onboarding(e) {
+  // filtering out unwanted trigger activation. 
+  // https://stackoverflow.com/questions/54834837/how-can-i-be-getting-multiple-unwanted-event-blocks-from-the-same-onformsubmit-t/54860085#54860085
+  if (e.values && !e.values[1]) {
+    return;
```

Comment:
why not a console.log here so we can find out when this happens?

Reply:
Problem found: it's not because of Google, only because there were indeed multiple triggers in the document.